src/messageprocessor.ts

```diff
@@ -19,8 +19,22 @@ const ID_EMOJI_REGEX_GROUP = 3;
 
 marked.setOptions({
     sanitize: true,
+    tables: false,
 });
 
+const markedLexer = new marked.Lexer();
+for (const r of ["hr", "heading", "lheading", "blockquote", "list", "item", "bullet", "def", "table", "lheading"]) {
+    markedLexer.rules[r] = /$^/;
+}
+markedLexer.rules.paragraph = /^((?:[^\n]+\n\n)+)\n*/;
+
+const markedInlineLexer = new marked.InlineLexer(true);
+for (const r of ["tag", "link", "reflink", "nolink", "br"]) {
+    markedInlineLexer.rules[r] = /$^/;
+}
+markedInlineLexer.rules.em = /^_([^_](?:[^_]|__)*?[^_]?)_\b|^\*((?:\*\*|[^*])+?)\*(?!\*)/;
```

Comment:
Nit pick about 20-35 -> I'd prefer this to be in a ``function _setupMarked()`` and then called at the bottom of the file, only because I feel like it's a bit messy.